packages/commercetools/composables/src/useCart/index.ts

```diff
@@ -38,15 +39,16 @@ const params: UseCartFactoryParams<Cart, LineItem, ProductVariant, any> = {
     const updatedCart = await apiApplyCartCoupon(currentCart, coupon);
     return { updatedCart: updatedCart.data.cart, updatedCoupon: coupon };
   },
-  removeCoupon: async ({ currentCart, coupon }) => {
-    const updatedCart = await apiRemoveCartCoupon(currentCart, coupon);
+  removeCoupon: async ({ currentCart }) => {
+    const { id: couponId } = getCoupon(currentCart);
```

Comment:
what if we will change the way getter works? I don't think it's safe to use getters inside composable factories - they're prone to changes

Reply:
So how should it been done in your opinion? I thought it would be really good solution. Wdyt about it @andrzejewsky ?

Reply:
if you want to use the same logic in both geetter and here, just move it to the separated file / function and re-use in both places: one in getter, second here - because we can't use getters inside of composables, that breaks our api-design, getters are designed to be used in the themes, providing agnostic-way,